### ISSUE TITLE
Restrict search by location to visible bounds

### ISSUE THREAD

**Issue by lklepner** [nypl-spacetime/nypl-warper#36]
Searching for maps by location returns results which are not visible in the current location. It would be more accurate/useful if the results only contained maps which have visible content with the search bounds.

**Comment by mgiraldo** [comment]
search by location (i think) returns maps whose bounds intersect with the current map interface rectangle. is this not the case? one change could be to show maps whose bounds are contained by the interface. that would reduce the amount of results significantly, though.

**Comment by lklepner** [comment]
It seems the search by location is returning maps which don't have visible map (cropped image) within the search bounds. Some of the results are matching on the overscan or color guide, not the cropped area of the map.

**Comment by lklepner** [comment]
I suspect the search results are inaccurate since the clipmap_bounds is being saved incorrectly - https://github.com/nypl-spacetime/nypl-warper/issues/45

**Comment by mgiraldo** [comment]
yeah. @bertspaan is working on tools that surface problematic maps (no/wrong crop, no bbox, etc). i just started working on saving the map rectify and resampling modes.

**Comment by lklepner** [comment]
awesome. i'll keep focusing on UI refinements.
